geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/process/tube/TubeSelectProcessTest.scala

```diff
@@ -31,8 +31,7 @@ class TubeSelectProcessTest extends Specification {
 
   sequential
 
-  val dtgField = org.locationtech.geomesa.accumulo.process.tube.DEFAULT_DTG_FIELD
```

Comment:
it was only used in test classes, so i thought it was ok to remove... i think there are already constants for 'dtg' we could use instead, let me check.